public/pages/AnomalyCharts/containers/AnomaliesChart.tsx

```diff
@@ -183,7 +188,11 @@ export const AnomaliesChart = React.memo((props: AnomaliesChartProps) => {
         <EuiFlexGroup direction="column">
           {props.isHCDetector &&
           props.onHeatmapCellSelected &&
-          props.detectorCategoryField ? (
+          props.detectorCategoryField &&
+          (props.showAlerts !== true ||
+            (props.showAlerts &&
+              props.onDisplayOptionChanged &&
+              props.entityAnomalySummaries)) ? (
```

Comment:
Could you abstract this out to some boolean var for readability and to understand what this means exactly?

Reply:
sure. will do that.